platform/migration/platform-migratable-api/src/main/java/org/codice/ddf/migration/ImportMigrationException.java

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.migration;
+
+/**
+ * Exception that indicates some problem when importing the system' configurations.
+ * <p>
+ * <b>
+ * This code is experimental. While this interface is functional
+ * and tested, it may change or be removed in a future version of the
+ * library.
+ * </b>
+ * </p>
+ */
+public class ImportMigrationException extends MigrationException {
+    private static final String MESSAGE_PREFIX = "Failed to import configurations";
+
+    /**
+     * Constructs a new exception with the specified detail message prefixed with a common import
+     * configuration error.
+     *
+     * @param message the detail message. The detail message is saved for later retrieval by the
+     *                {@link #getMessage()} method.
+     */
+    public ImportMigrationException(String message) {
+        super(String.format("%s: %s", ImportMigrationException.MESSAGE_PREFIX, message));
```

Comment:
I prefer to prefix all static methods and attributes with the class name. It makes it clearer. The compiler takes care of all of that anyway.